corehq/toggles.py

```diff
@@ -81,7 +80,7 @@ def randomness_percent(self):
         return "{:.0f}".format(self.randomness * 100)
 
     def _get_identifier(self, item):
-        return '{}:{}:{}'.format(self.namespace, self.slug, item)
+        return '{}:{}:{}'.format(self.namespaces, self.slug, item)
```

Comment:
funny side effect is that the first key of the hash is just converting the list to it's default string representation which is something like `"['domain', 'user']"`. It might be more clear to join these as explicit strings, but that would change the current list of toggle-enabled domains unpredictably (ha!). not sure if we care about that though since this is basically just for canary releasing anyways.